frontend/javascripts/oxalis/shaders/segmentation.glsl.ts

```diff
@@ -166,22 +166,22 @@ export const convertCellIdToRGB: ShaderModule = {
   `,
 };
 // This function mirrors the above convertCellIdToRGB-function.
-// Output is in [0,1] for H, S, L and A
-export const jsConvertCellIdToHSLA = (
+// Output is in [0,1] for R, G, B, and A
+export const jsConvertCellIdToRGBA = (
   id: number,
-  customColors?: Array<number> | null | undefined,
+  customColors?: Array<Vector3> | null | undefined,
```

Comment:
None of the callers uses this parameter which is why I changed its format. I think it's purely there to "mirror" the functionality of the `convertCellIdToRGB` shader function as pointed out in the comment above. Since the `customColors` are in RGB format in the shader as well, I figured it's fine to mirror that, here.